modules/order/order.graphql.js

```diff
@@ -0,0 +1,26 @@
+const orderType = `
+type Order {
+  _id: ID!
+  status: String
+  user: OrderUser,
+  dateOfCreation: String,
+  delivery: Delivery,
+  items: [OrderItems],
+  totalPrice: [CurrencySet]
+  paymentMethod: String
+}`
+const orderInput = ` 
+input OrderInput {
+  status: String
```

Comment:
We also need fields:
- `comment` that reflects comment from a customer about some specifics related to the order
- `shouldCall` that reflects does the client want to be called by admin after making the order

Reply:
fixed